include/robot_localization/ros_filter.hpp

```diff
@@ -245,6 +245,31 @@ class RosFilter : public rclcpp::Node
   //!
   void periodicUpdate();
 
+  //! @brief Update filter with data from measurements queue.
+  //! @param[in] time - The time at which to carry out integration.
+  //!
+  void updateFilterWithMeasurements(const rclcpp::Time & time);
+
+  //! @brief publish world to base link transform and position
```

Comment:
Would this go over 100 characters without the line break?

Reply:
Yep. Fixed.